Archer v2: Fixed fetch incident time + upload file to record (#10035)
demisto/content: Packs/ArcherRSA/Integrations/ArcherV2/README.md

2. Search for RSA Archer v2.
3. Click **Add instance** to create and configure a new integration instance.

## Clarifications  

> The time_zone (offset) parameter is used if the CortexXSOAR's and Archer's server aren't in the same time zone.

Reviewer: ```suggestion
The timezone (offset) parameter should be used if the Cortex XSOAR server and Archer's server aren't in the same time zone.
```

> If the CortexSOAR server time is 00:00 and the Archer server time is 01:00, the time_zone parameter should be -60 (minutes).

Reviewer: ```suggestion
If the Cortex XSOAR server time is 00:00 and the Archer server time is 01:00, the timezone parameter should be -60 (minutes).
```
